docs/formatter.md

```diff
@@ -392,3 +392,15 @@ flag.
 Black promotes some of its preview styling to stable at the end of each year. Ruff will similarly
 implement formatting changes under the [`preview`](https://docs.astral.sh/ruff/settings/#preview)
 flag, promoting them to stable through minor releases, in accordance with our [versioning policy](https://github.com/astral-sh/ruff/discussions/6998#discussioncomment-7016766).
+
+## Sorting imports
+
+Currently, the Ruff formatter does not sort imports. In order to both sort imports and format,
+call the Ruff linter and then the formatter:
+
+```shell
+ruff check --select I --fix .
+ruff format .
+```
+
+A unified command for both linting and formatting is currently [being developed](https://github.com/astral-sh/ruff/issues/8232).
```

Comment:
We're not actively working on it just yet (we hopefully get to it at the beginning of next year). I'm not sure if it matters or if we should change the sentence to *linting and formatting is planned.

Reply:
I rephrased that part accordingly, what do you think?